File: Vista/ClientApp/src/app/modelos/Usuarios.ts
import { Data } from "@angular/router";

export interface Usuarios {
  UsuarioId: number;
  CorreoElectronico: string;
  Usuario: string;
  Contrasena: string;
  Estatus: boolean;
  Sexo: number;
  FechaCreacion: Data;
}

File: Entidades/Usuarios.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Entidades
{
    public class Usuarios
    {
        public int UsuarioId { get; }
        public string CorreoElectronico { get; set; }
        public string Usuario { get; set; }
        public string Contrasena { get; set; }
        public bool Estatus { get; set; }
        public int Sexo { get; set; }
        public DateTime FechaCreacion { get; set; }
    }
}
